irohad/torii/query_client.cpp

```diff
@@ -11,6 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include "network/impl/grpc_channel_builder.hpp"
 #include "torii/query_client.hpp"
```

Comment:
Reorder please, first include in .cpp file should be the .hpp that you implement